packages/untp-test-suite/integration/cli/untpTest.integration.test.ts

```diff
@@ -0,0 +1,228 @@
+import fs from 'fs';
+import { exec } from 'child_process';
+import { ConfigContent } from '../../build/core/types';
+
+describe("CLI 'untp test' Commands", () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  describe('process test runner and final report return PASS', () => {
+    let credentialFileName: string;
+    let storePath: string;
+    let stdout: any;
+    let credentials: ConfigContent[];
+
+    const mockPath = `${process.cwd()}/integration/mock/untpTestPass`;
+    beforeAll((done) => {
+      credentialFileName = 'credentialsExample.json';
+      storePath = `${mockPath}/${credentialFileName}`;
+      const fileContent = fs.readFileSync(storePath, 'utf8');
+
+      try {
+        credentials = JSON.parse(fileContent).credentials;
+      } catch (error) {
+        expect(error).toBeNull();
+      }
+
+      exec(`yarn untp test -c ${storePath}`, (error, result) => {
+        if (error) {
+          console.error(`execSync error: ${error}`);
+          return;
+        }
+
+        stdout = result;
+        done();
+      });
+    });
+
+    describe('Validation of `schema` in the configuration file', () => {
+      it('should ensure that the schema file of each credential exists', () => {
+        expect(stdout).not.toBeNull();
+        for (const credential of credentials) {
+          expect(
+            fs.existsSync(`${process.cwd()}/src/schemas/${credential.type}/${credential.version}/schema.json`),
+          ).toBe(true);
+        }
+      });
+
+      it('should return the content of `schema` when file is valid', () => {
+        expect(stdout).not.toBeNull();
+        for (const credential of credentials) {
+          const data = fs.readFileSync(
+            `${process.cwd()}/src/schemas/${credential.type}/${credential.version}/schema.json`,
+            'utf8',
+          );
+
+          expect(data).not.toBe('');
+        }
+      });
```

Comment:
This case should be outside the PASS case, it should separate to another. Because in order to run the `untp test` command, you need to pass this test case.